From 8fedf598c85f2fc5c68c96eebdc5753a89be7137 Mon Sep 17 00:00:00 2001
From: Pavel Raiskup <praiskup@redhat.com>
Date: Thu, 16 Jun 2016 10:12:12 +0200
Subject: [PATCH 1/7] bootstrap: fix bootstrap after clean clone

* bootstrap.conf: Reintroduce changes added by
c7e60954dc7945635194 and removed by cleanup in
0396591026410f9.
* lib/Makefile.am: Honor the ^pax prefix.
* po/POTFILES.in: Likewise.
---
 bootstrap.conf  | 12 +++++++++++-
 lib/Makefile.am |  8 ++++----
 po/POTFILES.in  |  6 +++---
 3 files changed, 18 insertions(+), 8 deletions(-)

diff --git a/bootstrap.conf b/bootstrap.conf
index fe4248f..403e371 100644
--- a/bootstrap.conf
+++ b/bootstrap.conf
@@ -124,4 +124,14 @@ do
 done
 
 copy_files ${PAXUTILS}/paxlib lib pax
-	  
\ No newline at end of file
+
+echo "$0: Creating lib/sysdep.c"
+m4 -DMODE=C sysdep.m4 mingw.m4 > lib/sysdep.c
+echo "$0: Creating lib/sysdep.h"
+m4 -DMODE=H sysdep.m4 mingw.m4 > lib/sysdep.h
+echo "$0: Updating lib/system.h"
+mv lib/system.h lib/system.h~
+cat lib/system.h~ > lib/system.h
+echo '#include "sysdep.h"' >> lib/system.h
+echo "$0: Creating am/sysdep.m4"
+m4 -DMODE=AC sysdep.m4 mingw.m4 > am/sysdep.m4
diff --git a/lib/Makefile.am b/lib/Makefile.am
index b8dd2b2..1353098 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -30,12 +30,12 @@ AM_CPPFLAGS = -I$(top_srcdir)/gnu -I../ -I../gnu
 
 noinst_HEADERS = system.h system-ioctl.h rmt.h paxlib.h
 libpax_a_SOURCES = \
- exit.c\
- exit-status.c\
+ paxexit.c\
+ paxexit-status.c\
  paxlib.h\
  rtapelib.c\
  sysdep.h\
  sysdep.c\
  system.h\
- error.c\
- names.c
+ paxerror.c\
+ paxnames.c
diff --git a/po/POTFILES.in b/po/POTFILES.in
index 8b4d7f7..2bad67d 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -27,9 +27,9 @@ gnu/quotearg.c
 gnu/version-etc.c
 gnu/xalloc-die.c
 
-lib/error.c
-lib/exit.c
-lib/names.c
+lib/paxerror.c
+lib/paxexit.c
+lib/paxnames.c
 lib/rtapelib.c
 
 rmt/rmt.c
-- 
2.20.1

